### PR TITLE
2.27.0

### PR DESCRIPTION
**Pull request by rasika-chivate**
Merged two PR's for 2.27.0 release

### CODE REVIEW

**Comment by denisok** [comment]
image
when I click on the left panel and try to choose 2.27.0 release notes

https://pmm-doc-pr-725.onrender.com/release-notes/2.27.0.md

File: mkdocs-base.yml [hunk 1]
```diff
@@ -235,6 +235,7 @@ nav:
   - faq.md
   - Release Notes:
       - release-notes/index.md
+      - "PMM 2.27.0": release-notes/2.27.0.md"
```
Comment (denisok) [inline mkdocs-base.yml:238]:
```suggestion
      - "PMM 2.27.0": release-notes/2.27.0.md
```

**Review by denisok — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.27.0.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+### PMM and Percona Portal Integration
+
+- **Enhanced UX for connecting PMM server to Percona Portal**
+
+    To leverage Percona Platform's recent support for federated identity, PMM now uses access-token authorization for connecting PMM instances to Percona Platform. 
+
+    This replaces the former username/password authentication used in PMM 2.26.0 and older versions.
+
+    For more information, see [Integrate PMM with Percona Platform](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/integrate-platform.md).
```
Comment (denisok) [inline docs/release-notes/2.27.0.md:36]: is this correct link ? `integrate-platform.md` at least it doesn't work on rendered preview.

File: docs/release-notes/2.27.0.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+    This replaces the former username/password authentication used in PMM 2.26.0 and older versions.
+
+    For more information, see [Integrate PMM with Percona Platform](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/integrate-platform.md). 
+
+-  **Access to account information for Percona customers**
+  
+      When you connect your PMM instances to Percona Platform as a Percona customer, you can now check all your Percona Platform account information on the new **Entitlements** and **Support Tickets** tab on the main menu.      
+  
+    For more information, see [Check your account information](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/account-info.md) in the online Help.
```
Comment (denisok) [inline docs/release-notes/2.27.0.md:42]: same here, broken link for me.

File: docs/release-notes/2.27.0.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/release-notes/2.27.0.md:36]: @catalinaadam could you please check.

File: docs/release-notes/2.27.0.md (hunk 3 above)
Comment (rasika-chivate) [reply docs/release-notes/2.27.0.md:42]: @catalinaadam Please check...

File: docs/release-notes/2.27.0.md (hunk 2 above)
Comment (catalinaadam) [reply docs/release-notes/2.27.0.md:36]: @denisok, @rasika-chivate these are links to topics that got added after this branch was created. They are available with the Online Help changes  in https://github.com/percona/pmm-doc/pull/756

Comment (denisok) [reply docs/release-notes/2.27.0.md:36]: or it should be a relative ling to internal `md` and that would be rendered and converted to `html` ?

Comment (rasika-chivate) [reply docs/release-notes/2.27.0.md:36]: @denisok we normally provide HTML links in the Release Notes.

Comment (rasika-chivate) [reply docs/release-notes/2.27.0.md:36]: @catalinaadam Could you please change that...

Comment (catalinaadam) [reply docs/release-notes/2.27.0.md:36]: these are new topics so I would have to guess the HMTLs. 
Is it that important to have html topics in just particular topic Rel Notes topics, whereas we have .md references in all other ones?

Comment (rasika-chivate) [reply docs/release-notes/2.27.0.md:36]: @catalinaadam. It's not about guessing the HTML. I can help with the links.

Comment (catalinaadam) [reply docs/release-notes/2.27.0.md]: @denisok, you were right. Checked and fixed the links should be ok now

**Review by denisok — APPROVED** [review]

**Pull request merged**